Add logging documentation for Elixir SDK (#9700)
getsentry/sentry-docs: docs/platforms/elixir/configuration/logging.mdx

```elixir
Logger.error("This will be reported to Sentry")
```

## Logger Handler Configuration

You can configure the `:logger` handler under the `:config` key, as shown above. The configuration options are shown below.

<ConfigKey name="capture-log-messages">

When this is `true`, the handler will report all logged messages to Sentry (provided they're not filtered by `:excluded_domains` and `:level`). Defaults to `false`, which will only send crash reports, which are messages with metadata that has the shape of an exit reason and a stacktrace.

</ConfigKey>

<ConfigKey name="level">

the minimum [`Logger` level](https://hexdocs.pm/logger/Logger.html#module-levels) to send events for. Defaults to `:error`. You can lower this to `:warning`, for example, to also report `Logger.warning/1` calls.

</ConfigKey>

<ConfigKey name="metadata">

Use this to include non-Sentry logger metadata in reports. If it's a list of keys, metadata in those keys will be added in the `:extra` context, under the `:logger_metadata` key. If set to `:all`, all metadata will be included. Defaults to `[]`.

</ConfigKey>

> <ConfigKey name="excluded-domains">
> 
> Any messages with a domain in the configured list will not be sent. Defaults to `[:cowboy]` to avoid double-reporting events from `Sentry.PlugCapture`.
> 

Reviewer: ```suggestion
<ConfigKey name="excluded-domains">

This configuration option makes it so that messages with a domain in the configured list will not be sent. It's set to  `[:cowboy]` by default to avoid double-reporting events from `Sentry.PlugCapture`.

```

</ConfigKey>

<ConfigKey name="rate-limiting">

> Configures rate-limiting for reported logged messages to Sentry. This can help protect from reporting too many messages in a short time. By default, rate-limiting is disabled.
> 

Reviewer: ```suggestion
This configures rate-limiting for reported logged messages to Sentry and is disabled by default. It can help limit the number of messages being reported in a short period of time. 

```
